basic-features.md

```diff
@@ -1,80 +1,136 @@
+| Multi schema change: add column(s)                                                                       | Y            | Y            | N            | N            | N            | N            |
+| [Change column type](/sql-statements/sql-statement-modify-column.md)                                      | Y            | N            | N            | N            | N            | N            |
+| **Transactions**                                                                                         | **5.1**      | **5.0**      | **4.0**      | **3.1**      | **3.0**      | **2.1**      |
+| [Async commit](/system-variables.md#tidb_enable_async_commit-new-in-v50)                                 | Y            | Y            | N            | N            | N            | N            |
+| [1PC](/system-variables.md#tidb_enable_1pc-new-in-v50)                                                   | Y            | Y            | N            | N            | N            | N            |
+| [Large transactions (10GB)](/transaction-overview.md#transaction-size-limit)                             | Y            | Y            | Y            | N            | N            | N            |
+| [Pessimistic transactions](/pessimistic-transaction.md)                                                  | Y            | Y            | Y            | Y            | Y            | Experimental |
+| [Optimistic transactions](/optimistic-transaction.md)                                                    | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Repeatable-read isolation (snapshot isolation)](/transaction-isolation-levels.md)                       | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Read-committed isolation](/transaction-isolation-levels.md)                                             | Y            | Y            | Y            | N            | N            | N            |
+| **Partitioning**                                                                                         | **5.1**      | **5.0**      | **4.0**      | **3.1**      | **3.0**      | **2.1**      |
+| [Range partitioning](/partitioned-table.md)                                                              | Y            | Y            | Y            | Y            | Y            | N            |
+| [Hash partitioning](/partitioned-table.md)                                                               | Y            | Y            | Y            | Y            | Y            | N            |
+| [List partitioning](/partitioned-table.md)                                                               | Experimental | Experimental | N            | N            | N            | N            |
+| [List COLUMNS partitioning](/partitioned-table.md)                                                       | Experimental | Experimental | N            | N            | N            | N            |
```

Comment:
@mjonss Can you please review this section (Partitioning)?

Reply:
Not sure if `EXCHANGE PARTITION` should be included (experimental since 5.0?). Otherwise LGTM.

Reply:
I'm going to add it. It's useful.